dropshot/src/api_description.rs

```diff
@@ -135,7 +137,24 @@ pub enum ApiEndpointParameterLocation {
 pub enum ApiEndpointParameterName {
     Path(String),
     Query(String),
-    Body,
+    Body(ApiEndpointBodyContentType),
```

Comment:
"name" was already probably a stretch. If you can think of a better identifier here I think that would be an improvement.

Reply:
`ApiEndpointParameterMetadata`?

Reply:
A nice catchall word! I'm good w/ it. Better than `ApiEndpointParameterThingamajigger`

Reply:
Fixed in 409e18a44093ddf6f7a9b0efa53f2f2ad75f38bd.